gradle/libs.versions.toml

```diff
@@ -37,6 +37,8 @@ androidx-activity = { module = "androidx.activity:activity", version.ref = "andr
 androidx-activity-compose = { module = "androidx.activity:activity-compose", version.ref = "androidx.activity" }
 androidx-appCompat = { module = "androidx.appcompat:appcompat", version = "1.6.1" }
 androidx-compose-compiler = "androidx.compose.compiler:compiler:1.5.4"
+androidx-compose-ui-test-junit4 = "androidx.compose.ui:ui-test-junit4:1.5.4"
+androidx-compose-ui-test-manifest = "androidx.compose.ui:ui-test-manifest:1.5.4"
```

Comment:
Please pull out a version entry for these as they are grouped: https://developer.android.com/jetpack/androidx/releases/compose-ui